great_expectations/execution_engine/sqlalchemy_data_splitter.py

```diff
@@ -39,7 +55,13 @@ def __eq__(self, other):
 
 
 class SqlAlchemyDataSplitter:
-    """Methods for splitting data accessible via SqlAlchemyExecutionEngine."""
+    """Methods for splitting data accessible via SqlAlchemyExecutionEngine.
+
+    Note, for convenience, you can also access DatePart via the instance variable
+    date_part e.g. SqlAlchemyDataSplitter.date_part.MONTH
+    """
+
+    date_part: DatePart = DatePart
```

Comment:
Any reason why you've opted for the class attribute instead of just using the enum as-is?

Reply:
Just so someone can do `from great_expectations.execution_engine.sqlalchemy_data_splitter import SqlAlchemyDataSplitter` and they don't also have to import DatePart. Just to clean up our interface + documented code examples. A little trick I learned from @donaldheppner. Which now I realize does not matter in this case since SqlAlchemyDataSplitter is not meant to be used outside of an execution engine or data connector... well, at least it can make our internal code cleaner.